### PR TITLE
feat: Add MaPicto

### PR DESCRIPTION
**Pull request by sergigraciaj**
Implement MaPicto following [Watt guidelines.](https://www.figma.com/file/7RhweKibaUnxX6WYBl7zKW/%E2%9A%A1%EF%B8%8F-Watt-Design-System?node-id=12983%3A12880)
Implementation is pretty similar as in MaIcon, svg's are loaded from [https://assets.holaluz.com/](https://assets.holaluz.com)/

### CODE REVIEW

**Review by alex-castelo — COMMENTED** [review]
Looks good to me! I think it's a good implementation. Good job Sergi 💪🏽

File: src/components/MaPicto/MaPicto.vue [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  },
+
+  computed: {
+    sizeInPx(): string {
+      const sizingProperties: Record<Size, string> = {
+        xsmall: '24',
+        small: '32',
+        medium: '48',
+        large: '64',
+        xlarge: '80',
```
Comment (edufarre) [inline src/components/MaPicto/MaPicto.vue:71]: Shouldn't this live in our tokens section?

File: package.json [hunk 2]
```diff
@@ -29,7 +29,8 @@
     "test:watch": "jest --watch",
     "tokens:build": "ts-node .style-dictionary/build.ts",
     "tokens:watch": "chokidar  'src/tokens/*.ts' -c 'npm run tokens:build'",
-    "vetur:build": "vue-int --output 'vetur' --input 'src/components' --recursive"
+    "vetur:build": "vue-int --output 'vetur' --input 'src/components' --recursive",
+    "postinstall": "npx simple-git-hooks"
```
Comment (edufarre) [inline package.json:33]: Why are we adding the postinstall script now? We did not tackle it when we port the project to TS? 🤔

File: src/components/MaPicto/MaPicto.vue (hunk 1 above)
Comment (sergigraciaj) [reply src/components/MaPicto/MaPicto.vue:71]: As this tokens are very specific to the MaPicto I think they have to live in the component pretty much as we did in MaIcon though sizes are different.

File: package.json (hunk 2 above)
Comment (sergigraciaj) [reply package.json:33]: Nope : (

**Review by edufarre — APPROVED** [review]

**Review by edufarre — COMMENTED** [review]
I did approve the PR, but I am missing some component tests :)

**Review by icurdu — APPROVED** [review]
LGTM! I added some comments, but for me it's ready to go!

File: src/components/MaPicto/MaPicto.stories.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    picto: pictos[0],
+  },
+  argTypes: {
+    picto: {
+      control: {
+        type: 'select',
+        options: pictos,
+      },
+    },
+    accent: {
```
[the remark this answers is not in the document]
Comment (sergigraciaj) [reply src/components/MaPicto/MaPicto.stories.js:18]: IMO tone is for the whole component as we do in alerts or icons, but we can change it to tone, I'm ok with that!

File: docs/components/MaPicto.docs.mdx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import { Meta } from '@storybook/addon-docs/blocks'
+
```
[the remark this answers is not in the document]
Comment (sergigraciaj) [reply docs/components/MaPicto.docs.mdx:2]: What u mean? Something like this?

**Pull request merged**